loki/transformations/single_column/annotate.py

```diff
@@ -221,14 +266,16 @@ def device_alloc_column_locals(cls, routine, column_locals):
             routine.body.prepend((ir.Comment(''), pragma, ir.Comment('')))
             routine.body.append((ir.Comment(''), pragma_post, ir.Comment('')))
 
-    def annotate_driver_loop(self, loop):
+    def annotate_driver_loop(self, loop, acc_vars):
         """
         Annotate driver block loop with ``'openacc'`` pragmas.
 
         Parameters
         ----------
         loop : :any:`Loop`
             Driver :any:`Loop` to wrap in ``'opencc'`` pragmas.
```

Comment:
Not in this PR, but since you're here maybe fix the typo: `openacc` instead of `opencc` ;-)